src/main/java/com/dropwizard/example/h2/DAO/PersonDAO.java

```diff
@@ -0,0 +1,35 @@
+package com.dropwizard.example.h2.DAO;
+
+import com.dropwizard.example.h2.mapper.PersonMapper;
+import com.dropwizard.example.h2.model.Person;
+import org.jdbi.v3.sqlobject.config.RegisterRowMapper;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.customizer.BindBean;
+import org.jdbi.v3.sqlobject.statement.GetGeneratedKeys;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface PersonDAO {
+
+    @SqlQuery("select * from person where id = :id")
+    @RegisterRowMapper(PersonMapper.class)
+    Optional<Person> getById(@Bind("id") Long id);
+
+    @SqlQuery("select id, firstname, lastname from person")
```

Comment:
In line 17 you have wild (*) select and in Line 21 you listed all columns explicitly. Any reason behind that?